src/esp/scene/SceneNode.h

```diff
@@ -90,10 +90,10 @@ class SceneNode : public MagnumObject {
   const Magnum::Range3D& getCumulativeBB() const { return cumulativeBB_; };
 
   //! set local bounding box for meshes stored at this node
-  void setMeshBB(Magnum::Range3D meshBB) { meshBB_ = std::move(meshBB); };
+  void setMeshBB(Magnum::Range3D meshBB) { meshBB_ = meshBB; };
 
   //! set the global bounding box for mesh stored in this node
-  void setAbsoluteAABB(Magnum::Range3D aabb) { aabb_ = std::move(aabb); };
+  void setAbsoluteAABB(Magnum::Range3D aabb) { aabb_ = aabb; };
```

Comment:
Interesting.  I guess these copies are cheaper to perform than the moves?  Impressive that clang-tidy is smart enough to tease that out.

Reply:
All `Magnum::Math` types are trivially copyable, there's no move constructor that `std::move()` could use.

Related -- for perf reasons it might be worth to pass it via `const Magnum::Range3D&`, it's 9 floats after all, so not exactly small anymore. (Personally I have a rule that anything that's two pointers or less is fine to copy by value, everything else should be a `const&`.)